### PR TITLE
AS: add new API to add a child to an existing resource

### PR DESCRIPTION
**Pull request by mistydemeo**

### CODE REVIEW

File: src/archivematicaCommon/lib/archivesspace/client.py [hunk 1]
```diff
@@ -601,3 +601,36 @@ def add_digital_object(self, parent_archival_object, dashboard_uuid, title="", i
             "digital_object": {"ref": new_object_uri}
         })
         self._post(parent_archival_object, data=json.dumps(parent_record))
+
+    def add_child(self, parent, title="", level=""):
```
Comment (Hwesta) [inline src/archivematicaCommon/lib/archivesspace/client.py:642]: Does this need to be added to ATK to maintain API compatibility?

**Comment by Hwesta** [comment]
:+1:

File: src/archivematicaCommon/lib/archivesspace/client.py (hunk 1 above)
Comment (mistydemeo) [inline src/archivematicaCommon/lib/archivesspace/client.py:642]: It does; I'll look into doing the equivalent there.

**Comment by mistydemeo** [comment]
Thinking about this, maybe this should be `PATCH` and `PUT` should be reserved for new records? Or should `PUT` handle both?

**Comment by mistydemeo** [comment]
Updated to use POST for new records.

**Pull request merged**